### DISCUSSION TITLE
Move angular project to its own submodule

### DISCUSSION THREAD

**Discussion by wnleao** [sevdokimov/log-viewer#28] [Ideas]
Would it be okay to move the angular project to its own submodule like logviewer-frontend? That way we are decoupling the component and isolating the angular project build from the java core/libraries.

**Comment by sevdokimov** [comment]
yes, that's right

**Reply by wnleao** [reply]
Okay! Do you want me to pull request it or are you going to do it?

**Reply by sevdokimov** [reply]
I have no time. If you can do it, it would be great.

**Reply by wnleao** [reply]
Okay, I'll create an issue

**Reply by wnleao** [reply]
I'm not able to manage the issue that I created. Could you give me permission later?

**Reply by sevdokimov** [reply]
How to do it? I could not found settings for managing issues.

**Reply by wnleao** [reply]
Take look at this: https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository

I've issued the pull request. If you have any problem let me know.
